cmd/asset-syncer/postgresql_utils.go

```diff
@@ -112,38 +110,22 @@ func (m *postgresAssetManager) UpdateLastCheck(repoName, checksum string, now ti
 }
 
 func (m *postgresAssetManager) importCharts(charts []models.Chart) error {
-	txn, err := m.DB.Begin()
```

Comment:
Why are you also removing the transaction for importing the charts?

Reply:
I forgot to explain that. The only supported method when doing a transaction is `COPY FROM` (https://godoc.org/github.com/lib/pq#hdr-Bulk_imports). That was the reason for a chart to be added several times to the database. Once I set the unique column, that method started to fail because "there was already an entry with that unique ID".

I tried to use a custom query (instead of COPYing) but arguments were not supported. The only way I found to safely do this was to execute one query per chart (and it didn't have a noticeable impact in the performance).